### PR TITLE
SDSS-56 event arc prototype

### PR DESCRIPTION
**Pull request by dgnball**
Create a Cloud Function that is initiated by uploading a new object to a storage bucket. The function will read the object out of storage and store it in Firebase.

### CODE REVIEW

**Review by martyncolmer — APPROVED** [review]

File: src/app/main.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    event_id = cloud_event["id"]
+    event_type = cloud_event["type"]
+
+    bucket_name = data["bucket"]
+    filename = data["name"]
+    metageneration = data["metageneration"]
+    time_created = data["timeCreated"]
+    updated = data["updated"]
+
+    print(f"Event ID: {event_id}")
```
Comment (tidmanmike) [inline src/app/main.py:20]: Remove prints

File: src/app/main.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import database
+import dataset_storage
+import functions_framework
+
+
+# Triggered by a change in a storage bucket
+@functions_framework.cloud_event
+def new_dataset(cloud_event):
+    data = cloud_event.data
```
Comment (tidmanmike) [inline src/app/main.py:9]: Doc Strings

File: src/app/main.py (hunk 1 above)
Comment (dgnball) [reply src/app/main.py]: Done

File: src/app/main.py (hunk 2 above)
Comment (dgnball) [reply src/app/main.py:15]: Done

**Review by tidmanmike — APPROVED** [review]

**Pull request merged**